File: Corzbank.Data.Entities/Models/CardModel.cs
using Corzbank.Data.Enums;
using System;
using System.Collections.Generic;
using System.Text;

namespace Corzbank.Data.Entities.Models
{
    public class CardModel
    {
        public CardType CardType { get; set; }

        public PaymentSystem PaymentSystem { get; set; }

        public string SecretWord { get; set; }
    }
}

File: Corzbank.Web/ClientApp/src/app/data/models/card.model.ts
import { Guid } from "guid-typescript";
import { CardType } from "../enums/card-type.enum";
import { PaymentSystem } from "../enums/payment-system.enum";

export class CardModel{
    cardType:CardType;
    paymentSystem:PaymentSystem;
    secretWord:string;
    userId:string;
}